README.md

```diff
@@ -16,7 +16,14 @@ During the operation of the application, the following exceptions are possible:
 - errorCode 40007 - message: The resource already exists;
 - errorCode 40008 - message: Parameter is not filled or filled incorrectly;
 - errorCode 40009 - message: Unrecognized JSON field (not marked as ignorable);
-- errorCode 40010 - message: Incorrect sorting parameters (possible: by "name", "date")
+- errorCode 40010 - message: Incorrect sorting parameters (possible: by "name", "date");
+
+- errorCode 40101 - message: Unauthorized error: Full authentication is required to access this resource;
+- errorCode 40102 - message: Refresh token was expired. Please make a new login request;
```

Comment:
please do not provide explanation for authentication. just return smth like `incorrect token` for all cases